docs/get-started/cardano-node/installing-cardano-node.md

```diff
@@ -1,122 +1,146 @@
+Consider setting up the IOG binary cache in order to avoid building the universe locally on your machine. See the [IOGX](https://github.com/input-output-hk/iogx/blob/main/doc/nix-setup-guide.md) template documentation for more information.
 
-```
-conflict: pkg-config package libsodium-any, not found in the pkg-config database
-```
+### Building via `cabal`
 
-despite having `libsodium` installed in the system.
+To download the source code and build it, you need the following packages and
+tools on your system:
 
-The possible solutions to this problem are:
+* the version control system `git`,
+* a C and C++ compiler, either `gcc` or `clang`,
+* developer libraries for:
+  * the arbitrary precision library `gmp`,
+  * the compression library `zlib`,
+  * the service manager `systemd`,
+  * the TUI library `ncurses`,
+  * the key-value database `lmdb`,
+  * the cryptographic suite `openssl`,
+* `ncurses` compatibility libraries,
+* the Haskell build tool `cabal` (`3.10.2.0` or above),
+* the GHC Haskell compiler (version `9.6.7` or above).
 
-- to upgrade `pkgconf` to version 2 by building it from source:
-  ```
-  yum update -y && \
-  yum install git diffutils gcc gcc-c++ tmux gmp-devel make tar xz wget zlib-devel libtool autoconf -y && \
-  mkdir -p /root/src/ && cd /root/src && \
-  curl -O https://distfiles.ariadne.space/pkgconf/pkgconf-2.0.3.tar.xz && \
-  tar -xvf pkgconf-2.0.3.tar.xz && \
-  cd pkgconf-2.0.3 && \
-  ./configure && make && make install && \
+#### System libraries
+
+<div class="tabsblock">
+<Tabs>
+  <TabItem value="ubuntu" label="Debian/Ubuntu" default>
+    ```bash
+    sudo apt-get update -y
+    sudo apt-get install automake build-essential pkg-config libffi-dev libgmp-dev libssl-dev libncurses-dev libsystemd-dev zlib1g-dev make g++ tmux git jq wget libtool autoconf liblmdb-dev -y
+    ```
+  </TabItem>
+  <TabItem value="fedora" label="Fedora, RedHat or CentOS">
+    ```bash
+    sudo yum update -y
+    sudo yum install git gcc gcc-c++ tmux gmp-fdevel make tar xz wget zlib-devel libtool autoconf -y
+    sudo yum install systemd-devel ncurses-devel ncurses-compat-libs which jq openssl-devel lmdb-devel -y
+    ```
+  </TabItem>
+  <TabItem value="macos" label="MacOS">
+     You'll need the following packages and tools on your MacOS system:
+
+     * [Xcode](https://developer.apple.com/xcode) - The Apple Development IDE and SDK/Tools
+     * [Xcode Command Line Tools](https://developer.apple.com/xcode/features/), you can install it by typing `xcode-select --install` in the terminal.
+     * [Homebrew](https://brew.sh) - The Missing Package Manager for MacOS (or Linux)
+
+     Then using homebrew install the following:
+
+     ```bash
+     brew install jq libtool autoconf automake pkg-config openssl
+     ```
+
+     You will need to install llvm in case you are using M1
+
+     ```
+     brew install llvm@13
+     ```
+  </TabItem>
+  <TabItem value="windows" label="Windows MSYS2">
```

Comment:
Should we put an entry for windows MSYS2/WSL here if it's not a first class citizen upstream (or in community beyond Ddz wallet users, who anyways use packaged binary)?

Primarily concerned with futute maintenance of this section being up to date , which could fall off if there arent active users.

Also, hardening references for Windows will be vastly different on other pages

Reply:
I think it is worth having this entry however I agree with it getting out of date. Perhaps a disclaimer saying "Due to the user base for Windows being small these instructions might get out of date. We welcome PRs to update them if you see something is off"

Reply:
Sounds goof